getting_started_notebooks/basics/streamz_weblogs.ipynb

```diff
@@ -473,9 +516,86 @@
     "consumer_conf = {'bootstrap.servers': bootstrap_servers, 'group.id': 'custreamz', 'session.timeout.ms': 60000}\n",
     "\n",
     "stream = Stream.from_kafka_batched(topic, consumer_conf, poll_interval='1s', npartitions=1, asynchronous=True, dask=False)\n",
-    "final_output = stream.map(process_on_gpu).sink(print)\n",
+    "final_output = stream.map(process_on_gpu).sink(save_to_file)\n",
     "stream.start()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The cell above creates a consumer that is waiting to read and process messages based on the topic and configuration specified.\n",
+    "\n",
+    "Using the confluent_kafka library we can set up a producer that writes messages to this topic."
```

Comment:
> "The cell above creates a consumer that is waiting to read and process messages based on the topic and configuration specified.\n",
    "\n",
    "Using the confluent_kafka library we can set up a producer that writes messages to this topic."

Suggestion:
Typically, upstream applications produce messages to Kafka. For the sake of a self contained example you can experiment with, we'll use the [Confluent Kafka](https://anaconda.org/conda-forge/python-confluent-kafka) library to produce messages that our Streamz app immediately consumes.